packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -105,6 +106,19 @@ export class MirrorNodeClient {
             timeout: 10 * 1000
         });
 
+        //@ts-ignore
+        axiosRetry(axiosClient, {
+            retries: parseInt(process.env.MIRROR_NODE_RETRIES!) || 3,
+            retryDelay: () => {
+                const delay = parseInt(process.env.MIRROR_NODE_RETRY_DELAY!) || 500;
+                return delay;
+            },
+            retryCondition: (error) => {
+                return !error?.response?.status || error?.response?.status === 404 || error?.response?.status === 504;
+            },
+            shouldResetTimeout: true
```

Comment:
should we add an onRetry to at least log the retries?

Reply:
Trace log :)

Reply:
Added trace log 
Ex. 
`[Request ID: 9263aa9e-605e-4ad3-950e-33665935f381] Retry delay 500 ms`

Reply:
Should be trace